File: data/llama3_8B_ins_results/readable_format_HPEC/cpp/file000.cpp
#include <iostream>
#include <omp.h>

using namespace std;

int main() {
    int len = 100;
    int a[len][len];

    #pragma omp parallel for collapse(2)
    for (int i = 0; i < len; i++) {
        for (int j = 0; j < len; j++) {
            a[i][j] += 1;
        }
    }

    return 0;
}

File: data/deepseek_coder_base_6.7B/readable_format_HPEC/fortran/file000.f90
program DRB093_doall2_collapse_orig_no
    use omp_lib
    use DRB093
    implicit none

    integer :: len, i, j
    len = 100

    allocate (a(len,len))

    !$omp parallel do collapse(2)
    do i = 1, len
        do j = 1, len
            a(i,j) = a(i,j)+1
        end do
    end do
    !$omp end parallel do
end program
